server/src/events.mjs

```diff
@@ -301,10 +303,12 @@ function coreSendEvent(isBroadcast, ws, userId, method, result, msg, fSuccess, f
   try {
     if (  ! isBroadcast && !isRegisteredEventListener(userId, method) ) {
       logger.info(`${method} event not registered`);
+      eventErrorType = 'registrationError'
       fErr.call(null, method);
```

Comment:
Im not 100% sure where the .call() function comes from, but instead of exporting eventErrorType from events.mjs, would we be able to pass it into the call below? So it would look something like this

fErr.call(null, method, errorType);

If we are able to do this it would simplify the logic inside of the api file